install.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
```

Comment:
It seems you need to install openface - is this available as installable release version, e.g. sudo apt-get install openface? Otherwise, you need to write a wrapper for it, so that it becomes installed as a ROS package. We need to talk about this, because this install script cannot satisfy the dependencies in the ROS way and hence we cannot make a release with having this install file.

By the way, would the ROS package https://github.com/interaction-lab/openface_ros just wrap this software? Then you have an easy job and just need to depend on that one (so writing a new wrapper is not necessary then). I can help you to achieve this.